docs/debug.md

```diff
@@ -323,29 +308,28 @@ The most common culprits for sandbox issues are:
 * Attempting to use `/var/tmp`'. The program should be adjusted to use
   `TMPDIR` or `/tmp`
 * Attempting to use `/run`. The program should be adjusted to use
-  `SNAP_DATA` or `/run/shm/snaps/SNAP_FULLNAME/SNAP_VERSION`
-* Access denied to hardware. To fix for development, use
-  `snappy hw-assign <name> /dev/...`. When ready for production, consider
-  using a gadget snap.
-* Not specifying the correct `caps` for your snap (eg, not using
-  `network-service` for server software or declaring the cap to use a
-  particular framework service)
-* Trying to execute programs on the system or programs in `/snaps/bin`. Except
-  for a few common programs (eg, that are useful for shell programming),
+  `SNAP_DATA` or `/run/shm/snaps/SNAP_NAME/SNAP_REVISION`
+* Access denied to hardware. Such access is granted via interfaces-- make sure
+  you're using the correct one, and log a bug if an interface is missing
+  something you need.
+* Not specifying the correct interfaces for your snap (eg, not using
+  `network-bind` for server software)
+* Trying to execute programs on the system or programs in `/snap/bin`. Except
+  for a few common programs (e.g., that are useful for shell programming),
   apps should run programs from their application directory, not the system.
   In addition, apps should not try to run their programs installed in
-  `/snaps/bin`, but instead simply call them directly from `SNAP/...`
-  (executing from `/snaps/bin` doesn't work because they use the privileged
+  `/snap/bin`, but instead simply call them directly from `SNAP/...`
+  (executing from `/snap/bin` doesn't work because they use the privileged
   launcher to setup the sandbox, and apps aren't allowed to change their
   sandbox once they start)
 * A snap uses `setuid`/`setgid` or `chown` family of syscalls. Ubuntu Core
-  15.04 does not provide a mechanism of assigning users and groups to snaps,
-  so the `setuid`/`setgid` and `chown` family of syscalls are blocked (since
-  there is no appropriate user to change to. Optionally assigning
-  users/groups to snaps is a planned feature). For example,
+  16 does not provide a mechanism of assigning users and groups to snaps, so the
```

Comment:
I think it should be: a mechanism FOR assigning. But I might be wrong, it's your language, not mine :)